model/strand.rb

```diff
@@ -19,19 +19,34 @@ def subject
     UBID.decode(ubid)
   end
 
+  # :nocov:
+  schedule = if Config.development?
```

Comment:
Should we remove the `SCHEDULE` constant then?

Reply:
Yep, forgot about that.  I'll push a commit to remove it.  Probably also should only prepare the statement if it is needed.

Reply:
Pushed a commit for that.